README.md

```diff
@@ -118,6 +118,11 @@ another option is to use the `JsonWriter` to pipe output directly to somewhere e
 ```sh
 SOLR_URL=http://localhost:8983/solr/core-name bundle exec traject -c lib/traject/config/config_name.rb -w Traject::JsonWriter my_marc_file.marc | tail -n +2 | jq '.pub_country'
 ```
+when working with non-MARC data held locally (e.g. JSON exports from FOLIO), you can use the `FolioJsonReader` to pipe output into traject from stdin:
+```sh
+# after exporting OKAPI_URL, OKAPI_USER, OKAPI_PASSWORD
```

Comment:
Is this necessary? We aren't using the API, right?

Reply:
We never hit the API but I believe since folio_config constructs a client behind the scenes that's why we need it.
https://github.com/sul-dlss/searchworks_traject_indexer/blob/cc8768eb71e4bb586b67a3fa7664846559caddca/lib/traject/config/folio_config.rb#L36
When I run this command without providing those ENV vars, I get:

`Error loading configuration file lib/traject/config/folio_config.rb:37 URI::InvalidURIError:bad URI(is not URI?): nil`

Reply:
Perhaps we need another clause in the if statement there, so we don't build a client unless it is going to be used.

Reply:
Reworked the statement so that we don't try to build a client if we specified `FolioJsonReader` as the reader.